app/src/main/java/sun/trainingcourse/musicapp/data/model/Music.kt

```diff
@@ -0,0 +1,37 @@
+package sun.trainingcourse.musicapp.data.model
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Music(
+    val mSong: String?,
+    val mAuthor: String?,
+    val mPath: String?
```

Comment:
Ở kotlin thì đặt tên bình thường thôi 
```suggestion
    val song: String?,
    val author: String?,
    val path: String?
```

Reply:
Ok huy, quen dùng java nên hay đặt vậy